Sources/Metric Logs/MetricViews.swift

```diff
@@ -0,0 +1,297 @@
+                }
+            }
+
+            if let optimizedTimeToFirstDraw = payload.optimizedTimeToFirstDraw, !optimizedTimeToFirstDraw.buckets.isEmpty {
+                NavigationLink("Optimized Time To First Draw") {
+                    MetricView(title: "Optimized Time To First Draw", kind: .histogram(optimizedTimeToFirstDraw))
+                }
+            }
+        }
+        .navigationTitle("Launch")
+    }
+}
+
+struct LocationMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXLocationActivityMetric
+
+    // MARK: - View
+    var body: some View {
+        List {
+            MetricView(title: "Best for Navigation", kind: .measurement(metric.cumulativeBestAccuracyForNavigationTime))
+            MetricView(title: "Best", kind: .measurement(metric.cumulativeBestAccuracyTime))
+            MetricView(title: "Nearest 10 M", kind: .measurement(metric.cumulativeNearestTenMetersAccuracyTime))
+            MetricView(title: "Nearest 100 M", kind: .measurement(metric.cumulativeHundredMetersAccuracyTime))
+            MetricView(title: "Nearest KM", kind: .measurement(metric.cumulativeKilometerAccuracyTime))
+            MetricView(title: "Nearest 3 KM", kind: .measurement(metric.cumulativeThreeKilometersAccuracyTime))
+        }
+        .navigationTitle("Location Activity")
+    }
+}
+
+struct MemoryMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXMemoryMetric
+
+    // MARK: - View
+    var body: some View {
+        List {
+            MetricView(title: "Average Suspended Memory", kind: .average(metric.averageSuspendedMemory))
+            MetricView(title: "Peak Memory", kind: .measurement(metric.peakMemoryUsage))
+        }
+        .navigationTitle("Memory")
+    }
+}
+
+struct NetworkMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXNetworkTransferMetric
+
+    // MARK: - View
+    var body: some View {
+        List {
+            MetricView(title: "Cellular Down", kind: .measurement(metric.cumulativeCellularDownload))
+            MetricView(title: "Cellular Up", kind: .measurement(metric.cumulativeCellularUpload))
+            MetricView(title: "WiFi Down", kind: .measurement(metric.cumulativeWifiDownload))
+            MetricView(title: "WiFi Up", kind: .measurement(metric.cumulativeWifiUpload))
+        }
+        .navigationTitle("Network Transfer")
+    }
+}
+
+struct ResponsivenessMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXAppResponsivenessMetric
+
+    // MARK: - View
+    var body: some View {
+
+        if let hangTime = payload.hangTime {
+            MetricView(title: "Hang Time", kind: .histogram(hangTime))
+        } else {
+            NoEntriesView(configuration: .default)
+                .navigationTitle("Responsiveness")
```

Comment:
Can move navigation title out of the else.

Reply:
Moved it into an enclosing `Group`